tests/lib/update_tpp_schema.py

```diff
@@ -141,6 +141,12 @@ def apply_schema_modifications(by_table):
         {"ColumnName": "CodingSystem", "ColumnType": "int"},
     ]
 
+    # TODO: Remove this when the "OpenSAFELY-TPP database schema" report is next
+    # released.
```

Comment:
suggestion: It's potentially worth opening an issue and adding it to the backlog, so it doesn't get forgotten about.